src/components/Grid/__tests__/Grid.spec.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import { propTests } from '../../../../tests/playwright';
+import {
+  GridForTest,
+  GridWithFixedCardsHeightForTest,
+  GridWithGridSpanForTest,
+  GridWithoutChildrenForTest,
+} from './Grid.story';
+import { columnPropTest } from './_propTests/columnsPropTest';
+import { rowsPropTest } from './_propTests/rowsPropTest';
+import { justifyItemsPropTest } from './_propTests/justifyItemsPropTest';
+import { justifyContentPropTest } from './_propTests/justifyContentPropTest';
+import { alignContentPropTest } from './_propTests/alignContentPropTest';
+import { columnGapPropTest } from './_propTests/columnGapPropTest';
+import { rowGapPropTest } from './_propTests/rowGapPropTest';
+import { autoFlowPropTest } from './_propTests/autoFlowPropTest';
+
+test.describe('Grid', () => {
+  test.describe('base', () => {
+    test.describe('visual', () => {
+      [
```

Comment:
Again, please review the snapshots and setup tests in such way that we can see the real difference. If you compare alignemt left/center/space-beetween/end, it all loooks the same. For such tests, we need to see the difference .